.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
 
 # External Contributor Checklist
 
-🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
+🚨 Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.
```

Comment:
Will this work? The problem I think is that we need the full path, including this `blob/master` part: https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md

Reply:
Because if you look at it, this relative path is solved correctly, but as you are at https://github.com/mendersoftware/mender/pull/smthing still needs the `blob/master` bit

Reply:
It seems you might be right.
I was kinda thinking GitHub would be smart enough to resolve this...
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes

Reply:
Me too